difflint/lib.py

```diff
@@ -213,19 +213,37 @@ def main():
                         'PATH. If some are missing, reports which ones.')
     args = parser.parse_args()
 
-    missing_files = get_missing_linters()
+    missing_configurations = get_missing_configuration_files()
+    
+    missing_linters = []
+
+    # We cannot find the missing_linters if any required configuration files
+    # are missing.
+    if not missing_configurations:
+        missing_linters = get_missing_linters()
 
     if args.check:
-        if not missing_files:
+        if not missing_linters and not missing_configurations:
             print('All configuration files and linting programs found.')
             return 0
-        print(', '.join(missing_files) + ' not found.')
+        for linter_dict in missing_linters:
+            print('Linter "' + linter_dict['linter'] + ' for the language "' +
+                  linter_dict['language'] + '" is enabled but not found.')
+        for missing_configuration in missing_configurations:
+            print('Configuration file "' + missing_configuration + '"' +
+                  ' is missing.')
         return MISSING_FILE_EXIT_CODE
 
-    if missing_files:
-        stderr.write('Configuration and/or linting files missing. ' +
+    if missing_linters:
+        stderr.write('Required linting files missing. ' +
                      'Run `difflint --check` for a list of missing files.\n')
         return 0
+    
+    if missing_configurations:
```

Comment:
Ah, right. If we don't have a configuration, we won't have the linter information.